tensorflow/tools/api/golden/v1/tensorflow.pbtxt

```diff
@@ -1068,6 +1068,10 @@ tf_module {
     name: "cross"
```

Comment:
Now you can revert this file to make the API tests pass again

Reply:
Done. PTAL.